packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -31,17 +31,42 @@
 					:parameters="parametersNoneSetting"
 					:hideDelete="true"
 					:nodeValues="nodeValues" path="parameters" @valueChanged="valueChanged"
+					:isSupportedByHttpRequestNode="isSupportedByHttpRequestNode"
 				>
+					<n8n-notice
```

Comment:
might be better to move all this logic into its own component here `scopes-notice`

Reply:
This all was trimmed down very much with the refactored notice component:

```
<n8n-notice
	v-if="isHttpRequestNodeV2(node) && scopes.length > 0"
	:content="scopesShortContent"
	:fullContent="scopesFullContent"
/>
```

Is it worth it to extract still?

Reply:
Addressed in refactoring branch: https://github.com/n8n-io/n8n/pull/3304